announcements/2021/week09.md

```diff
@@ -0,0 +1,29 @@
+# :zap: This week in provider-independent announcements (March 5, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed apps
+
+[NGINX IC v1.15.0](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v1.15.0/) upgrades the container images (controller container to v0.44.0 and kube-webhook-certgen to v1.5.1). It also removes conflicting admission webhook api versions, ensuring the tree of apiversions will be valid according to the webhook.
+
+[Kong v1.2.0](https://docs.giantswarm.io/changes/managed-apps/kong-app/v1.2.0/) upgrades to upstream chart v1.15.0 and Kong to v2.3. Upstream chart v1.15.0 is an interim release before the planned v2.0.0 release. Please review the [upstream changelog](https://github.com/giantswarm/kong-app/blob/master/helm/kong-app/CHANGELOG.md) for all changes between v1.12.0 and v1.15.0.
+
+[Loki v0.1.1-alpha](https://docs.giantswarm.io/changes/managed-apps/loki-app/v0.1.1-alpha/) adds annotation to route alerts to the apps team, in preparation for offering Loki as a managed app in production.
+
+## Apps supported with best-effort
+
+[Jaeger Operator v0.2.3](https://docs.giantswarm.io/changes/playground-apps/jaeger-operator-app/v0.2.3/) upgrades to upstream v0.2.3.
```

Comment:
@ced0ps what is the right version? there's nothing in changelog 🙈  and in the PR it says v2.19.1 https://github.com/giantswarm/jaeger-operator-app/pull/30/files

but in v0.2.2 https://github.com/giantswarm/jaeger-operator-app/blob/master/CHANGELOG.md#022---2021-01-20, chart is 1.21.2 and jaeger-all-in-one is 1.21.0? What is 2.19.1? 🙏